packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -272,6 +284,30 @@ abstract class RenderViewportBase<ParentDataClass extends ContainerParentDataMix
     markNeedsLayout();
   }
 
+  /// This value is set during layout based on the [CacheExtentStyle].
+  ///
+  /// When the style is [CacheExtentStyle.viewport], it is the main axis extent
+  /// of the viewport
```

Comment:
Maybe worth pointing out that this is always expressed in pixels no matter the strategy?